### ISSUE TITLE
Any plans to split this into a separate library?

### ISSUE THREAD

**Issue by grtcdr** [uttarayan21/mctl-rs#1]
Hey uttarayan :)

I'd love to know if you plan to split mctl-rs into a separate library, so I (and potentially others) can use it in my own notifier program to get the status of mpd.

**Comment by uttarayan21** [comment]
Oh that seems like a good Idea I'll split the main into lib.rs and main.rs and keep everything in the same crate.

**Comment by grtcdr** [comment]
Thank you, I look forward to using it, let me know if you'd like any help.

**Comment by uttarayan21** [comment]
Sure. I'll let you know If I need help.

**Comment by grtcdr** [comment]
Off-topic, can I have a go at having `PlayerInfo` expose even more data, such as volume or state?

**Comment by grtcdr** [comment]
I'd also love to add some kind of way to notify the caller if the requested player is not running. Right now, if `Mpd` for example, is not running, the only thing you get back in the console is the string "Mpd", which is quite confusing.

**Comment by uttarayan21** [comment]
Yeah I actually just use It to bind the keys to the media keys in my keyboard so I didn't add It when I was writing. I should have added some descriptive errors/messages. You can go ahead and open a PR if you want.

**Comment by grtcdr** [comment]
Sure, I'm working on it right now :)
